Manifests/ManagedPreferencesApplications/com.github.macadmins.Nudge.plist

```diff
@@ -161,60 +188,271 @@
-			<key>pfm_documentation_url</key>
-			<string>https://github.com/macadmins/nudge/wiki/optionalFeatures#asyncronoussoftwareupdate&#45;&#45;&#45;type-boolean-default-value-true-required-no</string>
-			<key>pfm_macos_min</key>
-			<string>11.0</string>
-			<key>pfm_name</key>
-			<string>asyncronousSoftwareUpdate</string>
-			<key>pfm_title</key>
-			<string>Asyncronous Software Update</string>
-			<key>pfm_type</key>
-			<string>boolean</string>
-		</dict>
 		<dict>
-			<key>pfm_app_min</key>
-			<string>1.0</string>
-			<key>pfm_default</key>
-			<true/>
-			<key>pfm_description</key>
-			<string>When enabled, Nudge will enforce minor updates. This should likely never be disabled.</string>
+			<key>pfm_description_reference</key>
+			<string>All optional features to enhance and customize the Nudge experience for your employer's needs.</string>
 			<key>pfm_documentation_url</key>
-			<string>https://github.com/macadmins/nudge/wiki/optionalFeatures#enforceminorupdates&#45;&#45;&#45;type-boolean-default-value-true-required-no</string>
-			<key>pfm_macos_min</key>
-			<string>11.0</string>
+			<string>https://github.com/macadmins/nudge/wiki/Preferences#optionalfeatures---type-hash-default-value-</string>
 			<key>pfm_name</key>
-			<string>enforceMinorUpdates</string>
+			<string>optionalFeatures</string>
+			<key>pfm_subkeys</key>
+			<array>
+				<dict>
+					<key>pfm_app_min</key>
+					<string>1.1.4</string>
+					<key>pfm_default</key>
+					<true/>
+					<key>pfm_description_reference</key>
+					<string>When disabled, Nudge will not attempt to download a major upgrade application via softwareupdate.</string>
+					<key>pfm_documentation_url</key>
+					<string>https://github.com/macadmins/nudge/wiki/optionalFeatures#attempttofetchmajorupgrade---type-boolean-default-value-true-required-no</string>
+					<key>pfm_name</key>
+					<string>attemptToFetchMajorUpgrade</string>
+					<key>pfm_title</key>
+					<string>Attempt to Fetch Major Upgrade</string>
+					<key>pfm_type</key>
+					<string>boolean</string>
+				</dict>
+				<dict>
+					<key>pfm_app_min</key>
+					<string>1.1.7</string>
+					<key>pfm_description_reference</key>
+					<string>The application Bundle ID which Nudge allows without taking focus. (You can specify one or more Bundle ID.)
+
+Disables Nudge's re-activation logic for the listed applications. If one of the specified applications is foremost, Nudge will not reactivate itself. (This feature does not work for Nudge's initial activation.).</string>
+					<key>pfm_documentation_url</key>
+					<string>https://github.com/macadmins/nudge/wiki/optionalFeatures#acceptableapplicationbundleids---type-array-default-value--required-no</string>
+					<key>pfm_name</key>
+					<string>acceptableApplicationBundleIDs</string>
+					<key>pfm_subkeys</key>
+					<array>
+						<dict>
+							<key>pfm_name</key>
```

Comment:
As this is contained an array, the key name here should actually be `pfm_title`.  `pfm_name` is only used when we have a `<key></key>` and value, and as such wouldn't be in the case of these strings w/in the array.